client/me/notification-settings/reader-subscriptions/index.jsx

```diff
@@ -34,12 +34,10 @@ class NotificationSubscriptions extends Component {
 		return () => this.props.recordGoogleEvent( 'Me', 'Focused on ' + action );
 	}
 
-	handleCheckboxEvent( action ) {
+	handleCheckboxEvent( action, invert = false ) {
 		return ( event ) => {
-			const eventAction = 'Clicked ' + action + ' checkbox';
-			const optionValue = event.target.checked ? 1 : 0;
-
-			this.props.recordGoogleEvent( 'Me', eventAction, 'checked', optionValue );
+			const optionValue = invert ? ! event.target.checked : event.target.checked;
+			this.props.recordGoogleEvent( 'Me', `Clicked ${ action } checkbox`, 'checked', +optionValue );
```

Comment:
This kind of integer conversion is a bit more idiomatic, but I appreciate the brevity 👍